tests/test_cli.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import leo
+
+
+def test_cli():
+    args = leo.parse(["-l", "de", "apfel"])
+    print(args)
```

Comment:
Is that a art or can we remove that? :wink:

Reply:
oops, my bad